src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,6 +1,11 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
 public class CinemaHallRequestDto {
+    @Min(value = 0)
+    @Max(value = 300)
```

Comment:
let's remove max) I guess, it's possible that capacity could be greater than 300